requests-for-comments/2023-11-28-slack-removal.md

```diff
@@ -0,0 +1,31 @@
+# Request for comments: Closing the PostHog Users Slack
+
+This RFC is to explore the topic of potentially closing the PostHog Users Slack entirely. 
+
+## Argument: Why would we close it?
+
+The PostHog Users Slack currently exists in limbo. It was previously the main channel for getting customer feedback, chatting to users, fostering a community, and running community support. Now, however, we have better and more scalable methods for each of these - Surveys, user interviews, the Post/Questions section, and the support modal respectively. 
+
+As we've introduced each of these tools, we've moved further from the Slack to the point that the user Slack is no longer referenced or linked to anywhere on the site, and team members admit they rarely check it. 
+
+The same isn't true for users, a small ([but declining](/images/slack.png)) number of whom still use it almost universally for submitting support requests. Often these don't get a response from us (because it's not a support channel) or other users (because we've failed to foster a true community on Slack).
+
+The result is that the Slack gives users a bad experience, while providing little to no benefit to us or them. There's no clear owner to make any changes, and so the argument is that closing will improve the user experience with little to no impact for the team. 
```

Comment:
![image](https://github.com/PostHog/meta/assets/984817/fb851362-8448-4776-a8ab-9985d81dd15c)

I still think Slack is a better user interface than zendesk, but I don't have the energy to champion it. And I trust web/marketing to keep working on building community on other channels.

Reply:
And right now I agree... it's a ghost town and doesn't give a great impression